README.md

```diff
@@ -47,6 +47,11 @@ directly to a template repository. If `<LINK_TO_TEMPLATE_REPOSITORY>` proves to
 
 `dp template-list` lists all added templates.
 
+### Project update
```

Comment:
It would be nice to paste the same information in `docs/usage.rst` (changing the syntax accordingly). Also, please run `make apidoc` in `docs` directory (I have to come up with a GitHub Action check for this though) to add `update` to the documentation. It might require running `pip install -e .[docs]` before.